sections/performance/nativeoverutil.brazilian-portuguese.md

```diff
@@ -0,0 +1,70 @@
+# Prefira métodos nativos ao invés de utilitários do usuário como Lodash
+
+
+<br/><br/>
+
+### Explicação em um Parágrafo
+
+Às vezes, usar métodos nativos é melhor do que requerir `lodash` ou `underscore`, porque isso não levará a um aumento de desempenho e usará mais espaço do que o necessário.
+O desempenho usando métodos nativos resulta em um [ganho geral de 50%](https://github.com/Berkmann18/NativeVsUtils/blob/master/analysis.xlsx) que inclui os seguintes métodos: `Array.concat`, `Array.fill`, `Array.filter`, `Array.map`, `(Array|String).indexOf`, `Object.find`, ...
+
+
+<!-- compare aqui: https://gist.github.com/Berkmann18/3a99f308d58535ab0719ac8fc3c3b8bb-->
+
+<br/><br/>
+
+### Exemplo: comparação de benchmark - Lodash vs V8 (Nativo)
+O gráfico abaixo mostra a [média dos benchmarks para uma variedade de métodos do Lodash](https://github.com/Berkmann18/NativeVsUtils/blob/master/nativeVsLodash.ods), Isso mostra que os métodos Lodash levam em média 146,23% mais tempo para completar as mesmas tarefas que os métodos V8.
+
+![meanDiag](../../assets/images/sampleMeanDiag.png)
+
+### Exemplo de código – teste de Benchmark com `_.concat`/`Array.concat`
+```javascript
+const _ = require('lodash'),
+  __ = require('underscore'),
+  Suite = require('benchmark').Suite,
+  opts = require('./utils'); //cf. https://github.com/Berkmann18/NativeVsUtils/blob/master/utils.js
+
+const concatSuite = new Suite('concat', opts);
+const array = [0, 1, 2];
+
+concatSuite.add('lodash', () => _.concat(array, 3, 4, 5))
+  .add('underscore', () => __.concat(array, 3, 4, 5))
+  .add('native', () => array.concat(3, 4, 5))
+  .run({ 'async': true });
+```
+
+Que retornará isso:
+
+![output](../../assets/images/concat-benchmark.png)
+
+Você pode encontrar uma lista maior de benchmarks [aqui](https://github.com/Berkmann18/NativeVsUtils/blob/master/index.txt) ou alternativamente [executar isso](https://github.com/Berkmann18/NativeVsUtils/blob/master/index.js) que mostraria o mesmo porém com cores.
+
+### Citação de Blog: "Você não precisa (talvez não) de Lodash/Underscore"
```

Comment:
Wouldn't it be better if we use: 

> Você (talvez) não precisa de Lodash/Underscore

Reply:
Yes, that is better.

Reply:
@DouglasMV Since this is merged already, please open a new PR for the above suggestion (and also maybe link to this one).